### PR TITLE
Update installers for jdk11 to jdk-11.0.16+8

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Approval to merge during the lockdown cycle please

**Comment by zdtsw** [comment]
this is super interesting, all failed on `impish` with
```
			result = runShell(container, "DEBIAN_FRONTEND=noninteractive apt-get update");
			assertThat(result.getExitCode()).isEqualTo(0);
```
so impish container cannot do apt-get udpate?

**Comment by zdtsw** [comment]
re-run the previous GH action on "main" branch, 
does fail on `impish as` well https://github.com/adoptium/installer/runs/7542231916?check_suite_focus=true

**Review by karianna — COMMENTED** [review]
This looks find but the failing GH Actions are weird for sure

**Comment by gdams** [comment]
The impish tests are failing with this:

```output
INFO: Using parallel execution mode 'CONCURRENT' set via the 'junit.jupiter.execution.parallel.mode.default' configuration parameter.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
debconf: delaying package configuration, since apt-utils is not installed
debconf: delaying package configuration, since apt-utils is not installed
debconf: delaying package configuration, since apt-utils is not installed
E: The repository 'http://security.ubuntu.com/ubuntu impish-security Release' does not have a Release file.
E: 
The repository 'http://archive.ubuntu.com/ubuntu impish Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu impish-updates Release' does not have a Release file.
E: The repository 'http://archive.ubuntu.com/ubuntu impish-backports Release' does not have a Release file.
debconf: delaying package configuration, since apt-utils is not installed
debconf: delaying package configuration, since apt-utils is not installed
debconf: delaying package configuration, since apt-utils is not installed
debconf: delaying package configuration, since apt-utils is not installed
```

**Comment by gdams** [comment]
And I can confirm that the impish base image is broken:

```bash
➜  linux git:(July) docker run -it ubuntu:impish /bin/bash                          
root@798ae1fa0f55:/# apt-get update 
Ign:1 http://ports.ubuntu.com/ubuntu-ports impish InRelease
Ign:2 http://ports.ubuntu.com/ubuntu-ports impish-updates InRelease
Ign:3 http://ports.ubuntu.com/ubuntu-ports impish-backports InRelease
Ign:4 http://ports.ubuntu.com/ubuntu-ports impish-security InRelease
Err:5 http://ports.ubuntu.com/ubuntu-ports impish Release
  404  Not Found [IP: 185.125.190.36 80]
Err:6 http://ports.ubuntu.com/ubuntu-ports impish-updates Release
  404  Not Found [IP: 185.125.190.36 80]
Err:7 http://ports.ubuntu.com/ubuntu-ports impish-backports Release
  404  Not Found [IP: 185.125.190.36 80]
Err:8 http://ports.ubuntu.com/ubuntu-ports impish-security Release
  404  Not Found [IP: 185.125.190.36 80]
Reading package lists... Done
E: The repository 'http://ports.ubuntu.com/ubuntu-ports impish Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ports.ubuntu.com/ubuntu-ports impish-updates Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ports.ubuntu.com/ubuntu-ports impish-backports Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ports.ubuntu.com/ubuntu-ports impish-security Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Just checking if any bugs have been filed.

**Comment by gdams** [comment]
And there's our answer... Impish went EOL on the 14th July. @sophia-guo I'll raise a pull request to remove impish and then you'll need to rebase your PR's

File: linux/jdk/debian/src/main/packaging/temurin/11/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+temurin-11-jdk (11.0.16.0.0+8) UNRELEASED; urgency=medium
```
Comment (gdams) [inline linux/jdk/debian/src/main/packaging/temurin/11/debian/changelog:1]:
```suggestion
temurin-11-jdk (11.0.16.0.0+8) stable; urgency=medium
```

Looks like we should have switched this to stable some time ago. Let's do it from this release onwards.

**Comment by sophia-guo** [comment]
redhat 8 failed:

❌ rockylinux:8
	org.opentest4j.AssertionFailedError:

**Comment by sophia-guo** [comment]
rerun faild check job redhat8, passed.

**Review by gdams — APPROVED** [review]

**Pull request merged**